shaders/ScreenFilter.shader

```diff
@@ -1,29 +1,29 @@
 shader_type canvas_item;
 uniform int mode = 0;
 void fragment() {
-	vec4 px = texture(SCREEN_TEXTURE, SCREEN_UV);
```

Comment:
Just curious, all you did here was remove some tabs, the code is virtually identical, why does github make these changes look like the code was completely replaced?

Reply:
From the POV of git every single line was changed.